src/scss/partials/_nav.scss

```diff
@@ -189,9 +189,13 @@ $nav-popover-padding               : spacing(x2);
                 }
             }
         }
-            .c-nav-list-text-sub {
-                display: block;
-            }
+        .c-nav-list-text-sub {
```

Comment:
are there no helpers in fozzie that allow you to do this?

Reply:
There isn't an ellipsis helper as far as I can see